package.json

```
@@ -2,8 +2,14 @@
   "name": "create-shared-react-context",
   "version": "1.0.3",
   "description": "A memoized wrapper of React.createContext",
-  "main": "index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
+  },
```

We want to retain the old entrypoints in addition to the new `exports` key for the most compatibility, since older versions of tools may not support exports. We still need `exports` though because newer versions of tools might only support `exports`:

```suggestion
  "main": "./dist/index.js",
  "types": "./dist/index.d.ts",
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "default": "./dist/index.js"
    }
  },
```